CRM/Utils/Check/Component.php

```diff
@@ -47,24 +78,49 @@ public function isEnabled() {
    *
    * @return array
    *   [CRM_Utils_Check_Message]
+   *
+   * @throws \API_Exception
+   * @throws \Civi\API\Exception\UnauthorizedException
    */
   public function checkAll() {
     $messages = [];
     foreach (get_class_methods($this) as $method) {
-      if ($method !== 'checkAll' && strpos($method, 'check') === 0) {
+      $r = $this->isDisabled($method);
```

Comment:
@eileenmcnaughton what is this used for at all?

Reply:
@seamuslee001 thanks - fixed